internal/gtfs/refresh_live_config_test.go

```
@@ -0,0 +1,95 @@
+		firstTick     = make(chan struct{})
+		sawAdded      = make(chan struct{})
+	)
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		switch r.URL.Path {
+		case "/initial.zip":
+			firstTickOnce.Do(func() { close(firstTick) })
+		case "/added.zip":
+			addedOnce.Do(func() { close(sawAdded) })
+		}
+		// #nosec G104
+		w.Write(bundle)
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)